test/api/v3/unit/libs/amazonPayments.test.js

```diff
@@ -3,33 +3,43 @@ import * as amzLib from '../../../../../website/src/libs/api-v3/amazonPayments';
 import amazonPayments from 'amazon-payments';
 var User = require('mongoose').model('User');
 
-describe('amazonPayments', () => {
-  beforeEach(() => {
-  });
-
-  describe('#getTokenInfo stubbed', () => {
+describe.only('amazonPayments', () => {
+  describe('stubbed', () => {
     let thisToken = 'this token info';
     let amzOldConnect;
 
     beforeEach(() => {
       amzOldConnect = amazonPayments.connect;
       amazonPayments.connect = () => {
-        let api = { getTokenInfo: (token, cb) => {
-          return cb(undefined, thisToken);
-        } };
-        return { api };
+        let api = {
```

Comment:
can't you stub the amazon payments library that use promises instead of stubbing the ones that use callbacks and we don't use directly (so we may be missing some error in our own implementation). Also for stubbing you should use sinon.js (http://sinonjs.org/) not replace objects directly